aws-janitor/resources/asg.go

```diff
@@ -103,12 +106,14 @@ func (AutoScalingGroups) ListAll(opts Options) (*Set, error) {
 }
 
 type autoScalingGroup struct {
-	ID   string
-	Name string
+	Account string
+	Region  string
+	ID      string
+	Name    string
 }
 
 func (asg autoScalingGroup) ARN() string {
-	return asg.ID
```

Comment:
I think this one is already using the ARN (see L41), it's just named improperly in the struct.

Reply:
true! thanks for pointing out

thanks for your review @ixdy 
PTAL